repo/packages/K/kubernetes/3/package.json

```diff
@@ -11,7 +11,7 @@
     "icon-large": "https://downloads.mesosphere.com/kubernetes/artifacts/images/k8s-256x256.png"
   },
   "tags": ["mesosphere", "framework", "pod", "label", "paas"],
-  "preInstallNotes": "In order for Kubernetes to start successfully all resources must be available in the cluster including ports, CPU shares and RAM.\nWe recommend a minimum of 1 node with 1 CPU share and 1 GB of RAM available for use by the Kubernetes service.\nNote that the service is alpha and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.",
+  "preInstallNotes": "In order for Kubernetes to start successfully all resources must be available in the cluster including ports, CPU shares and RAM.\nWe recommend a minimum of 1 node with 1 CPU share and 1 GB of RAM available for use by the Kubernetes service.\nNote that the service is alpha and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.\n\nWARNING: The standard configuration launches an integrated etcd WITHOUT PERSISTENCE.\n         For production follow http://docs.mesosphere.com/services/kubernetes/#install\n         to configure kubernetes with a highly available etcd cluster.\n",
```

Comment:
Maybe it's because I'm too close to the impl, but I find use of the word
'persistence' confusing and would rather see the limitation explained in
terms of failover capability.

On Tue, Dec 8, 2015 at 11:06 AM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In repo/packages/K/kubernetes/3/package.json
> <https://github.com/mesosphere/multiverse/pull/58#discussion_r46972712>:
>
> > @@ -11,7 +11,7 @@
> >      "icon-large": "https://downloads.mesosphere.com/kubernetes/artifacts/images/k8s-256x256.png"
> >    },
> >    "tags": ["mesosphere", "framework", "pod", "label", "paas"],
> > -  "preInstallNotes": "In order for Kubernetes to start successfully all resources must be available in the cluster including ports, CPU shares and RAM.\nWe recommend a minimum of 1 node with 1 CPU share and 1 GB of RAM available for use by the Kubernetes service.\nNote that the service is alpha and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.",
> > +  "preInstallNotes": "In order for Kubernetes to start successfully all resources must be available in the cluster including ports, CPU shares and RAM.\nWe recommend a minimum of 1 node with 1 CPU share and 1 GB of RAM available for use by the Kubernetes service.\nNote that the service is alpha and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.\n\nWARNING: The standard configuration launches an integrated etcd WITHOUT PERSISTENCE.\n         For production follow http://docs.mesosphere.com/services/kubernetes/#install\n         to configure kubernetes with a highly available etcd cluster.\n",
>
> Easy way to prove: scale the kubernetes app down to 0 and up to 1 in
> Marathon. BOOM. Two frameworks registered.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/multiverse/pull/58/files#r46972712>.
>